### ISSUE TITLE
Is there any way to test this module locally

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/sharethis#26]
I understand that this module requires an account at https://sharethis.com. However, before going live I was hoping to test the module on my local dev setup and I'm not seeing anything. 

I assume that this is probably because it can not connect with the site (https://sharethis.com), but it's not clear. 

If in fact, you can not do anything with this module unless you are on a server, it might be helpful to put a note in the readme. Either way, it might make sense to put a note about what you can or can not do in your local test environment to save future developers time trying to figure that out. 

I'm not sure if the module is broke, my environment is broke, or it's just that I can't expect anything to happen.

**Comment by stpaultim** [comment]
Someone just reported the same problem in the forum:
https://forum.backdropcms.org/forum/module-sharethis-doesnt-work

It might be the same problem. I am not sure if they are working locally or on a server with a domain that will register on sharethis.com

**Comment by alfabuster** [comment]
> Someone just reported the same problem in the forum:

Yes. It's me. I reported. Maybe I do something wrong, but in the source code of page where I installed these buttons have, but they are invisible...

Check anybody source code of the link https://nloinfo.com/posts/luna-2019-god-nlo
You can find `<div class="sharethis-inline-share-buttons"></div>`

**Comment by alfabuster** [comment]
Update. Interesting things happens. 2 my browsers: Firefox and Brave, don't showing this buttons from ShareThis, but when I open site with Chromium and Opera, and also mobile Firefox - it's working. Don't understand why Firefox and Brave hide these buttons...

Module seems like working but problem with visibility...
